src/Uno.UI.RemoteControl.VS/VSIXChannel/IUnoDevelopmentEnvironmentIndicator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Linq;
```

Comment:
These aren't needed too

Reply:
Good point, due to CI instability, will fix in coming PR